Parsers/ASimWebSession/Parsers/ASIMWebSessionIIS.yaml

```diff
@@ -84,13 +83,16 @@ ParserQuery: |
                 DstIpAddr = coalesce(ipv4_parts[0], ipv6_parts[0]),
                 DstPortNumber = coalesce(ipv4_parts[1], ipv6_parts[1], host_parts[1]),
                 HttpHost = coalesce (host_parts[0], HttpHost)
+        | extend
+               DstPortNumber = toint(DstPortNumber),
+               DstIpAddr = toint(DstIpAddr)
```

Comment:
"tostring" of course. Whlie at it, the conversions in lines 87 and 88 can be part of lines 84 and 85.